frontend/src/container/GuestUpload.jsx

```diff
@@ -75,13 +77,29 @@ class GuestUpload extends React.Component {
     }
   })
 
-  handleChangeFullName = e => this.setState({ guestName: e.target.value })
-  handleChangeComment = e => this.setState({ guestComment: e.target.value })
+  handleChangeFullName = e => this.setState({ guestFullname: { value: e.target.value, isInvalid: false } })
+  handleChangeComment = e => this.setState({ guestComment: e.target.value, isInvalid: false })
   handleChangePassword = e => this.setState({ guestPassword: { ...this.state.guestPassword, value: e.target.value } })
 
   handleAddFile = uploadFileList => {
```

Comment:
I'd rename the function parameter to something like `newFileList`
because currently, it's the same as the state and it's confusing